src/node_jsvmapi.cc

```diff
@@ -204,7 +204,7 @@ namespace v8impl {
     public:
       CallbackWrapperBase(const T& cbinfo)
         : _cbinfo(cbinfo),
-          _cbdata(cbinfo.Data().As<v8::Object>()) {
+          _cbdata(v8::Local<v8::Object>::Cast(cbinfo.Data())) {
```

Comment:
Oh, I see your note in the PR description about this. I think there is a better way to fix this, using the [template disambiguator](http://stackoverflow.com/questions/4077110/template-disambiguator) syntax I haven't used before:
```
cbdata(cbinfo.template Data().As<v8::Object>())
```